datasets/varen/varen.json

```diff
@@ -0,0 +1,164 @@
+{
+  "type": "dataset",
+  "id": "Varen",
+  "title": "Varen",
+  "description": "Inforatie over de over de pleziervaart en het ligplaatsen register voor de pleziervaart.",
+  "dateCreated": "2020-08-01T00:00:00",
+  "dateModified": "2020-08-01T00:00:00",
+  "license": "Creative Commons, Naamsvermelding",
+  "homepage": "https://data.amsterdam.nl",
+  "language": "nl",
+  "status": "beschikbaar",
+  "version": "0.0.1",
+  "objective": "Programma varen heeft de verplichting om openbare informatie waaronder op en afstap plaatsen te publiceren voor de bewoners van Amsterdam en de ketenpartners.",
+  "temporalUnit": "nvt",
+  "contactPoint": {
+    "name": "",
+    "email": "datapunt@amsterdam.nl"
+  },
+  "accrualPeriodicity": "maandelijks",
+  "spatialDescription": "Gemeente Amsterdam",
+  "theme": "Verkeer",
+  "publisher": "OIS",
+  "owner": "Gemente Amsterdam, Toezicht en Handhaving Openbare Ruimte",
+  "authorizationGrantor": "Toezicht en Handhaving Openbare Ruimte",
+  "keywords": "ligplaatsenregister, varen, opstaplocatie, afstaplocatie, grachten, varen",
+  "hasBeginning": "2020-01-01T00:00:00",
+  "crs": "EPSG:28992",
+  "tables": [
+    {
+      "id": "var_opafstapplaats",
+      "type": "table",
+      "provenance": "var_opafstapplaats",
+      "title": "var_opafstapplaats",
```

Comment:
De title wordt getoond in de api.data.amsterdam.nl/v1/docs/ als tekst die begrijpelijk is voor breed geïnteresseerden.
Zie bijvoorbeeld: https://api.data.amsterdam.nl/v1/docs/datasets/touringcars.html
Het is daarom wenselijk om in de Title op dataset of table niveau een meer functionele tekst neer te zetten.